config/initializers/date_time.rb

```diff
@@ -0,0 +1,2 @@
+# Date
+Date::DATE_FORMATS[:default] = '%d/%m/%Y'
```

Comment:
We already have time_date_formats.rb intializer, so better to add that there. Didn't know that this is the way to change defualt format.

Reply:
I'll move it!